hasura/metadata/databases/default/tables/public_circle_share_tokens.yaml

```diff
@@ -0,0 +1,53 @@
+table:
+  name: circle_share_tokens
+  schema: public
+object_relationships:
+- name: circle
+  using:
+    foreign_key_constraint_on: circle_id
+insert_permissions:
+- permission:
+    backend_only: false
+    check:
+      circle:
```

Comment:
`deleted_at:  _is_null: true` for circle

Reply:
addressed